src/BlogPostPage.jsx

```diff
@@ -87,20 +298,183 @@ if (blog) {
     return <Widget src="${REPL_ACCOUNT}/widget/Posts.ModeratedPostData" props={{ ...dataProps, renderPost }} />;
   };
 
+  const postUrl = `https://near.org/near/widget/BlogPostPage?accountId=${props.accountId}&blockHeight=${props.blockHeight}`;
+  const markdownObj = parseMarkdown(blog.blogContent);
+
+  const renderComment = (a) => {
+    return (
+      <div key={JSON.stringify(a)}>
+        <Widget
+          src={`${REPL_ACCOUNT}/widget/Comments.Comment`}
+          props={{
+            accountId: a.account_id,
+            blockHeight: a.block_height,
+            content: a.content,
+            highlight:
+              a.account_id === props.highlightComment?.accountId &&
+              a.block_height === props.highlightComment?.blockHeight,
+            GRAPHQL_ENDPOINT,
+            notifyAccountId,
+            item,
+          }}
+        />
+      </div>
+    );
+  };
+
+  const renderedComments = blog.blogComments?.map(renderComment);
+
   return (
     <>
-      <Widget
-        src="${REPL_ACCOUNT}/widget/DIG.Button"
-        props={{
-          label: "Back",
-          href: `/${REPL_ACCOUNT}/widget/Blog.Feed`,
-          iconLeft: "ph ph-arrow-left",
-          variant: "secondary",
-          size: "small",
-        }}
-        style={{ marginBottom: "2em" }}
-      />
-      <Widget src="${REPL_ACCOUNT}/widget/Moderation.CheckPostModeration" props={{ ...props, renderData }} />;
+      <BlogPostWrapper>
+        <Widget
+          src="${REPL_ACCOUNT}/widget/DIG.Button"
+          props={{
+            label: "Back To All Posts",
+            href: `/${REPL_ACCOUNT}/widget/Blog.Feed`,
+            iconLeft: "ph ph-arrow-left",
+            variant: "secondary",
+            size: "small",
+          }}
+          style={{ marginBottom: "1rem" }}
+        />
+        {/* RENDER BLOG HEADER IMAGE */}
+        <PostImage
+          imageUrl={
+            markdownObj[0].imageUrl || "https://pages.near.org/wp-content/uploads/2023/06/generic-green-blog.png"
+          }
+          alt="Post image"
+        />
+        <BlogPostContentWrapper>
+          <TitleSection>
+            {/* RENDER BLOG HEADER - TITLE   */}
+
+            <h1>{getFirstHeading(markdownObj)?.text || "Untitled"}</h1>
+
+            {/* RENDER BLOG HEADER - DATE */}
+
+            <p>
+              <Widget
+                src="${REPL_ACCOUNT}/widget/TimeAgo"
```

Comment:
@jackson-harris-iii can we easily show the non abbreviated date? like March 1, 2024? 
<img width="224" alt="Screen Shot 2024-03-12 at 9 31 38 AM" src="https://github.com/near/near-discovery-components/assets/1915116/50a14f8f-9ec2-41d2-bf42-a889028b645e">